mini-bank/transactions.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Transactions</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+    />
+    <link rel="stylesheet" href="../helpers/css/components.css" />
+    <link rel="stylesheet" href="mini-bank.css" />
+  </head>
+  <body>
+    <div class="container-fixed">
+      <section class="header-component">
+        <div class="navigations">
+          <div class="nav-buttons">
+            <ul>
+              <a href="transactions.html" class="active"
+                ><li>Transactions</li></a
+              >
+              <a href="deposits.html"><li>Deposit</li></a>
+              <a href="withdraw.html"><li>Withdraw</li></a>
+            </ul>
+          </div>
+
+          <a href="user-profile.html">
+            <div class="user-nav">
+              <div class="user-icon"><i class="fas fa-circle-user"></i></div>
+              <div id="user-name" class="user-name">Username</div>
+            </div>
+          </a>
+        </div>
+
+        <div class="balance-wrapper">
+          Bal: &nbsp
+          <div class="user-balance" id="user-balance">0.00</div>
+        </div>
+      </section>
+
+      <div class="page-desc-holder">
+        <h1 class="page-desc">Transactions</h1>
+      </div>
+
+      <div class="header-component">
+        <table>
+          <thead>
+            <tr>
+              <th>Timestamp</th>
+              <th>Tranx Ref</th>
+              <th>Type</th>
+              <th>Amount</th>
+              <th>Bal Before</th>
+              <th>Bal After</th>
+            </tr>
+          </thead>
+          <tbody id="transaction-table-body">
+            <td align="center" colspan="6">
+              <em>No transactions have been made!</em>
+            </td>
+            <!--Render feedback dynamically with JavaScript-->
+          </tbody>
+        </table>
+      </div>
+
+      <div class="create-account" id="create-new-account">
```

Comment:
This button shouldn't be here as the user already has an account. If they didn't, they won't be able to access this page.